packages/grz-pydantic-models/src/grz_pydantic_models/submission/metadata/v1.py

```diff
@@ -1290,39 +1292,79 @@ def check_duplicate_file_paths(self):
 
         return self
 
+    def determine_thresholds_for(
+        self,
+        donor: Donor,
+        lab_datum: LabDatum,
+    ) -> thresholds_model.Thresholds:
+        """
+        Determine the thresholds for a given combination of genomic study subtype, library type and sequence subtype.
```

Comment:
Minor suggestion: change this to say something more like "for a given donor and lab datum", since the thresholds now depend on more than genomic study, library, and sequence subtype.